.vscode/launch.json

```diff
@@ -0,0 +1,164 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "command": "npm start",
+            "name": "Launch web app",
+            "request": "launch",
+            "cwd": "${workspaceFolder}/cdot-web-app",
```

Comment:
This should be the newly named `webapp` directory instead of `cdot-web-app`

Reply:
updated